WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -732,8 +732,8 @@ extension UIImage {
 
     /// Shopping cart Purple
     ///
-    static var shoppingCartPurpleIcon: UIImage {
-        return UIImage(named: "icon-shopping-cart-purple")!
+    static var shoppingCartFilled: UIImage {
```

Comment:
Instead of listing the color on the icon name, I updated to make the difference between it and the other `shoppingCartIcon` clear. I also updated the icon to be a template icon by default, as we tint it with the accent color.